apps/publisher/extensions/assets/default/apis/assets.jag

```diff
@@ -68,34 +68,45 @@ require('/modules/publisher.js').exec(function (ctx) {
             var newAsset;
             if (uriMatcher.match(CREATE_URL)) {//to create asset
                 try {
-                    newAsset = assetApi.create(options, req, res, session);
-                    //If the asset is successfully created then return the asset
-                    if (newAsset) {
-                        response = responseProcessor.buildSuccessResponse(res,
-                                constants.STATUS_CODES.CREATED, newAsset)
+                    var contentTypeheader = req.getHeader('Content-Type');
+                    var n = contentTypeheader.indexOf("multipart/form-data");
```

Comment:
This could cause NPE. i.e. If client hasn't sent a content-type header. Please fix.